src/Features/Core/Portable/CodeRefactoringHelpers.cs

```diff
@@ -131,5 +171,42 @@ internal static class CodeRefactoringHelpers
 
             return TextSpan.FromBounds(start, end);
         }
+
+        /// <summary>
+        /// Strips leading and trailing whitespace from <paramref name="span"/>.
+        /// </summary>
+        /// <remarks>
+        /// Returns unchanged <paramref name="span"/> in case <see cref="TextSpan.IsEmpty"/>.
+        /// Returns empty Span with original <see cref="TextSpan.Start"/> in case it contains only whitespace.
+        /// </remarks>
+        private static async Task<TextSpan> GetStrippedTextSpan(
+            Document document,
+            TextSpan span,
+            CancellationToken cancellationToken
+            )
```

Comment:
1. place on previous line.
2. you did'nt really need to wrap these parameters.

Reply:
I was just following style of the file, though it's true that for these methods its unnecessary. Will fix.